tests/validation/cattlevalidationtest/core/test_k8_ingress.py

```diff
@@ -572,14 +572,24 @@ def test_k8s_ingress_11(client, kube_hosts):
     execute_kubectl_cmds(
         "replace ing --namespace="+namespace,
         expected_result, file_name=ingress_file_name_new)
-    wait_until_lb_ip_is_active(lb_ip[0], port_new, timeout=120)
+    lb_ip_updated = wait_for_ingress_to_become_active(ingress_name, namespace)
```

Comment:
There should  be no need for this call , since we have seen this to return the same Ip as before port edit.